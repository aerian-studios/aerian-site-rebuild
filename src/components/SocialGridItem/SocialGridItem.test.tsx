import { faVimeoV } from "@fortawesome/free-brands-svg-icons/faVimeoV";
import * as React from "react";

import * as renderer from "react-test-renderer";

import { SocialGridItem } from "./index";

describe("SocialGridItem", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <SocialGridItem
                    className="myClass"
                    icon={faVimeoV}
                    url="https://vimeo.com/aerianstudios"
                    text="aerian-studios"
                />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});
