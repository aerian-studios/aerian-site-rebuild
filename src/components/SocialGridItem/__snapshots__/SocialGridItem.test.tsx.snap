// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialGridItem renders correctly 1`] = `
<span
  className="myClass"
>
  <a
    aria-label="Visit Aerian studios on vimeo-v"
    href="https://vimeo.com/aerianstudios"
    rel="nofollow noreferer"
  >
    <i
      className="fa"
    />
    <p
      aria-hidden="true"
    >
      aerian-studios
    </p>
    <span
      aria-hidden="true"
    >
      View page
      <i
        className="fa"
      />
    </span>
  </a>
</span>
`;
